v22.1/security-reference/encryption.md

```diff
@@ -107,6 +99,33 @@ A few other recommendations apply for best security practices:
 Note that backups taken with the [`BACKUP`](../backup.html) statement **are not encrypted** even if Encryption at Rest is enabled. Encryption at Rest only applies to the CockroachDB node's data on the local disk. If you want encrypted backups, you will need to encrypt your backup files using your preferred encryption method.
 {{site.data.alerts.end}}
 
+## Encryption keys used by CockroachDB clusters
+
+To allow arbitrary rotation schedules and ensure security of the keys, CockroachDB uses multiple layers of keys:
+
+- **Store key**: A cluster's _store key_ is a _key encryption key (KEK) that CockroachDB uses to encrypt the cluster's data keys (see below).
+
+  For CockroachDB Self-Hosted clusters, you provide the store key and give its location to CockroachDB when starting the cluster. The store key file must contain 32 bytes (the key ID) followed by the key (16, 24, or 32 bytes). The size of the key dictates the version of AES to use (AES-128, AES-192, or AES-256). For an example showing how to create a store key, see [Generating Key Files](/docs/{{site.versions["stable"]}}/encryption.html#generating-store-key-files).
+
+  For {{ site.data.products.db }} clusters, the store key is created automatically when the cluster is created. For {{ site.data.products.dedicated }} clusters with [Customer-Managed Encryption Keys (CMEK)](/docs/cockroachcloud/cmek.html) enabled, the store key is encrypted with the CMEK key (see below) before being provided to cluster nodes.
```

Comment:
No, we don't use encryption-at-rest by default on Dedicated clusters.

Reply:
I'm confused comparing your comment to the text -- I don't think we assert that it's enabled by default, only when CMEK is enabled. Is it unclear?